react/rules/DEU.js

```diff
@@ -0,0 +1,59 @@
+import { isPastDate } from '../utils/dateRules'
```

Comment:
Missing changelog entry. Vide:

https://github.com/vtex/profile-form/blob/master/CHANGELOG.md#added

Reply:
hi @salesfelipe I have added the the changes to the changelog can you please verify?